### PR TITLE
[oap-native-sql] Enable mvn package to build native libs

### PR DESCRIPTION
**Pull request by JiayiChen785**
## What changes were proposed in this pull request?

Added the generate resource phase in spark columnar core to execute compile.sh and build the native library.

## How was this patch tested?

Manually tested.

### CODE REVIEW

File: oap-native-sql/cpp/compile.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+cd ${CURRENT_DIR}
+if [ -d build ]; then
+    rm -r build
+fi
+mkdir build
+cd build
+cmake ..
+make -j
+make -j
```
Comment (zhouyuan) [inline oap-native-sql/cpp/compile.sh:14]: looks like "make -j" will make trouble for travis tests

File: oap-native-sql/cpp/compile.sh [hunk 2]
```diff
@@ -10,8 +10,8 @@ fi
 mkdir build
 cd build
 cmake ..
-make -j
-make -j
+make
+make
 make install
```
Comment (zhouyuan) [inline oap-native-sql/cpp/compile.sh:17]: should remove "make install"

Comment (JiayiChen785) [reply oap-native-sql/cpp/compile.sh]: Removing this line passes the test, but will it cause problems as the script will no longer update lib files under system dir, such as libspark_columnar_jni.so

Comment (zhouyuan) [reply oap-native-sql/cpp/compile.sh]: It should be ok as this lib will be packaged in the jar file, and is there any special objective by "make" twice?

Comment (JiayiChen785) [reply oap-native-sql/cpp/compile.sh]: I cleaned the compile.sh and removed one "make". When running the compile bash script alone, it needs to "make" twice as the first failed after building protobuf. After added as a mvn stage, running "make" once is enough.

**Comment by zhouyuan** [comment]
@JiayiChen785 thanks for improving this!

**Pull request merged**